### PR TITLE
Leanplum level test coverage

### PR DESCRIPTION
**Pull request by hrishileanplum**
Test coverage for setUserId through Leanplum.h

### CODE REVIEW

File: LeanplumTests/LeanplumTests.m [hunk 1]
```diff
@@ -17,23 +21,65 @@ @interface LeanplumTests : XCTestCase
...
 }
 
-- (void)testPerformanceExample {
-    // This is an example of a performance test case.
-    [self measureBlock:^{
-        // Put the code you want to measure the time of here.
+/**
+ * Tests whether setting user attributes and id works correctly with API Call
+ */
+- (void) test_user_attributes_api_call
```
Comment (gg4race) [inline LeanplumTests/LeanplumTests.m]: seems the default naming convention is camel case, should we stay consistent?

File: LeanplumTests/LeanplumTests.m [hunk 2]
```diff
@@ -17,23 +21,65 @@ @interface LeanplumTests : XCTestCase
...
 
-- (void)testExample {
-    // This is an example of a functional test case.
-    // Use XCTAssert and related functions to verify your tests produce the correct results.
+/**
+ * Tests whether setting user attributes and id works correctly.
+ */
+- (void) test_user_attributes
+{
+    [LPTestHelper setupStub:200 withFileName:@"simple_success_response.json"];
```
Comment (gg4race) [inline LeanplumTests/LeanplumTests.m]: did you add this json file? I renamed the the one i made to simple_post_success....

File: LeanplumTests/LeanplumTests.m (hunk 1 above)
Comment (hrishileanplum) [reply LeanplumTests/LeanplumTests.m]: sure, thats a good point. Will update this.

File: LeanplumTests/LeanplumTests.m (hunk 2 above)
Comment (hrishileanplum) [reply LeanplumTests/LeanplumTests.m]: yea, i will need to rebase my code with master to get your new file names.

**Pull request merged**